doc/help/kdb-gen.md

```diff
@@ -0,0 +1,71 @@
+# kdb-gen(1) -- Elektra's code-generator
+
+## SYNOPSIS
+
+`kdb gen [options...] <templateName> <parentKey> <outputName> [parameters...]`
+
+- `[options...]`:
```

Comment:
This is now inconsistent with the other man pages (where the options are not mentioned here).